### ISSUE TITLE
Using Shared interface elements on the android L jump problem

### ISSUE THREAD

**Issue by HotBitmapGG** [chrisbanes/PhotoView#386]
Using Shared interface elements on the android L jump, there will be a picture display position at the top of the interface, the fingers after pictures will return to center position, this is what I use is wrong or what causes, his tried many ways to solve, hope to be able to reply, thank you

**Comment by MaxMustermann2-0** [comment]
I have the same issue. As a workaround I use this:
```
@Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_image_viewer);
        mImageView = (ImageView) findViewById(R.id.image_view);
        mImageView.setOnTouchListener(this);
    }
```
```
@Override
    public boolean onTouch(View view, MotionEvent motionEvent) {
        if(view == mImageView){
            mImageView.setOnTouchListener(null);
            PhotoViewAttacher attacher = new PhotoViewAttacher(mImageView);
            attacher.update();
            return true;
        }
        return false;
    }
```

**Comment by Jawnnypoo** [comment]
This has been resolved in 1.3.0
